typedoc.js

```diff
@@ -1,25 +1,8 @@
 module.exports = {
-  exclude: ['**/*spec.ts', './src/index.ts', './src/wasm/wasm_exec.js'],
+  exclude: ['**/testSetup/*', '**/*.spec.ts', './src/wasm/wasm_exec.js'],
   listInvalidSymbolLinks: true,
-  stripInternal: true,
+  excludeExternals: true,
   excludeNotExported: true,
   excludePrivate: true,
-  hideGenerator: true,
-  mode: 'file',
-  tsconfig: 'tsconfig.json',
```

Comment:
Why remove the explicit setting of the _tsconfig.ts_?

Reply:
not sure? Why did we add it? It's not in the SDK. But maybe I add it again. Not sure what the best way would be... 32ea26a5317cd917e5a3dae7130f647204410f84

Reply:
I guess it does not matter but it should not hurt to set it.